### ISSUE TITLE
Issue only with windows, but working on linux

### ISSUE THREAD

**Issue by Chaprnks** [openmultiplayer/open.mp#953]
**Describe the bug**
I am testing a default/stock installation of open.mp from the releases section. Everything goes smoothly on linux, loads the components, then displays the loading of the gamemode. When I load this on a windows host, it goes smoothly up until the end of the loading components. It hangs/freezes after "Loading _xx_ component(s) from _xx_", never loading anything past that.

**To Reproduce**
1. Download linux release on linux host
2. Load linux binary
3. Download windows release on windows host
4. Load windows binary

**Expected behavior**
Since both tests are identical (version, components, gamemodes etc), I would expect both to load exactly the same.

**Screenshots and/or logs**
### Linux:
![linux](https://github.com/user-attachments/assets/da9d5a4b-c1a4-402b-b64b-09d42383d3a7)

### Windows:
![windows](https://github.com/user-attachments/assets/ab73f6d1-5820-41fe-a6e5-5067b75c05cb)

**Release version**
[v1.2.0.2670](https://github.com/openmultiplayer/open.mp/releases/tag/v1.2.0.2670)

**Comment by AmyrAhmady** [comment]
What is your system locale set to?(or windows language)

**Comment by Chaprnks** [comment]
> What is your system locale set to?(or windows language)

English (United States)

**Comment by AmyrAhmady** [comment]
Can you try latest nightly?

**Comment by Chaprnks** [comment]
> Can you try latest nightly?

Unfortunately, it does the same thing.

![1](https://github.com/user-attachments/assets/e984c3d4-9f2e-4970-af74-18b0257014bc)
![2](https://github.com/user-attachments/assets/050297b6-e562-4b4a-985c-a34abf0a0e44)

**Comment by AmyrAhmady** [comment]
Is it possible to stay in touch on discord?

**Comment by Chaprnks** [comment]
> Is it possible to stay in touch on discord?

Yes, of course. My discord is chaprnks (they must have changed the discord username system, cause it doesn't show in the Name#ID format anymore)
